src/tests/ipc_tests.rs

```diff
@@ -0,0 +1,242 @@
+#[cfg(test)]
+pub mod ipc_tests {
+    use crate::interface;
+    use crate::safeposix::{cage::*, filesystem, dispatcher::*};
+    use super::super::*;
+    use std::os::unix::fs::PermissionsExt;
+    use std::fs::OpenOptions;
+    use std::time::Instant;
+
+    //#[test]
+    pub fn test_ipc() {
+        ut_lind_ipc_pipe();
+        ut_lind_ipc_domain_socket();
+        ut_lind_ipc_socketpair(); 
+    }
+
+    pub fn ut_lind_ipc_pipe() {
+
+        let byte_chunk: usize = 131072;
+        let num_writes: usize = 8192;
```

Comment:
Can we change these and the other values to their 2<<x representations? like 2<<17 and 2<<13